packages/mds-policy/schema-gen/NoParkingPolicySchema.json

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
I think that all of these new schemas should be deleted now, right?

Reply:
idk why it didn't delete them, thanks for noticing